docs/sources/whatsnew/whats-new-next/README.md

```diff
@@ -0,0 +1,57 @@
+# Contribute to 'What's New in Grafana Cloud'
+
+To have a feature presented in [What's New in Grafana Cloud](https://grafana.com/docs/grafana-cloud/whatsnew/), add an entry to the [`index.md`](./index.md) file in this directory.
+
+Use the following template, replace any `<VARIABLE>` with the appropriate text (explained after the template):
+
+```markdown
+## <FEATURE>
+
+<!-- <CONTRIBUTOR> -->
+<!-- <ON-PREMISE OFFERING> -->
+
+<CLOUD AVAILABILITY> in Grafana <CLOUD OFFERING>
+
+<DESCRIPTION>
+```
+
+## _`CONTRIBUTOR`_
+
+The name of the contributor of the feature.
+The information is intentionally commented out so that it isn't displayed in the published page.
+
+## _`ON-PREMISE OFFERING`_
+
+One or both of:
+
+- OSS
+- Enterprise
+
+Intended availability of the feature when released outside of Grafana Cloud.
+The information is intentionally commented out so that it isn't displayed in the published page.
```

Comment:
```suggestion
The information is intentionally commented out so that it isn't displayed in the published page.
If the feature is not going to be released outside of Grafana Cloud, omit the HTML comment entirely.
```